tool/cpu_sample_intervals.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'dart:io';
+
+/// Outputs the time intervals between adjacent cpu samples.
+/// 
+/// A json file path is required as a command line argument.
+/// Ex: dart cpu_sample_intervals.dart ~/Downloads/example_json.dart
```

Comment:
Ex: --> Example usage:
nit. example should be
`example.json`
not
`example_json.dart`